src/Sentry/SentrySdk.cs

```diff
@@ -614,9 +614,34 @@ public static void BindException(Exception exception, ISpan span)
     /// Gets the Sentry trace header.
     /// </summary>
     [DebuggerStepThrough]
+    // [Obsolete("This method will be removed in a future major version. Use the `GetTraceParent` instead.")]
```

Comment:
@bitsandfoxes I'm wondering why `GetTraceParent` is a better name than `GetTraceHeader`. It looks like this method is returning a trace header... that header does include the parentid, so I guess it's the parent. Maybe `GetParentTraceHeader` is a more explicit name for what this is then (vs the TraceHeader that we'll be propagating downstream)?

Reply:
I could not find `GetTraceHeader` in other SDKs so this is something unique to the .NET SDK. Right now I'm just following the unified API to keep things consistent with the requirements. But maybe we opt into reusing the method to avoid the "rename". 
Also, the function changes slightly: `GetTraceHeader` would return the trace header of the last active span or null if there is no present. Since there's now always the `PropagationContext` available on the scope `GetTraceParent` will always a trace header.

Reply:
also: https://www.w3.org/TR/trace-context/#traceparent-header

Reply:
Hm... that's even more confusing. Our `SentryPropagationContext` is propagating the `sentry-trace` header right (not the W3C `traceparent` header)?